backend/src/main/java/com/google/blackswan/mock/SimpleAlertGenerator.java

```diff
@@ -0,0 +1,57 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.blackswan.mock;
+
+import com.google.models.*;
+import java.util.*;
+import java.util.Collections;
+
+/** Generate list of alerts by grouping anomalies by their month. */
+public class SimpleAlertGenerator implements AlertGenerator {
+
+  private List<Alert> alerts;
+  private AnomalyGenerator anomalyGenerator;
+
+  public SimpleAlertGenerator(AnomalyGenerator anomalyGenerator) {
+    this.alerts = new ArrayList<Alert>();
+    this.anomalyGenerator = anomalyGenerator;
+    groupAnomaliesToAlerts();
```

Comment:
a possible alternative as well (for this specific implementation)

`this.alerts = groupAnomaliesToAlerts(anomalyGenerator);`

and then make `private static List groupAnomaliesToAlerts(AnomalyGenerator anomalyGenerator)`

but this will depend on the grouping ie if they will use more class variables, etc, which might make sense to keep the grouping function non-static.

Reply:
the only class variables `groupAnomaliesToAlerts` use is the `anomalyGenerator`, so i think it makes sense to use the alternative implementation and make the function `static`. 

just to double check, the advantages to having `groupAnomaliesToAlerts` be a `private static` function instead of the original implementation would be showing that the function is a pure function and will not have any side effects on instance variables of the class?

Reply:
Yep! I refactored the code :)